modules/ROOT/pages/_partials/branch-commit-push-changes.adoc

```diff
@@ -1,18 +1,19 @@
 // tag::create-branch-vcs[]
 To create a different branch in Studio:
 
-. In the Package Explorer view, right-click your API specification project and select *Team* > *Switch To* > *New Branch...*.
+. In the Package Explorer view, right-click your API specification project and select *Team* > *Switch To* > *New Branch*.
 +
-image::studio::switch-to-new-branch.png[]
+image::studio::switch-to-new-branch.png["New branch highlighted in the context menu".]
 . In the *Create Branch* window, configure your new branch:
 +
-image::studio::create-new-branch.png[]
+image::studio::create-new-branch.png["*Branch name* field, *Configure upstream for push and pull* option, and *Check out new branch* option highlighted in the *Create Branch* window".]
```

Comment:
Do we need to bold here since Alt text is read by a screen reader?

Reply:
I use bold to avoid labels to be translated. Thanks for you comment.